code/game/objects/effects/decals/cleanable/humans.dm

```diff
@@ -8,6 +8,12 @@
 	blood_state = BLOOD_STATE_HUMAN
 	bloodiness = MAX_SHOE_BLOODINESS
 
+/obj/effect/decal/cleanable/blood/Crossed(mob/user)
+	. = ..()
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		C.adjust_hygiene(-5)

```

Comment:
So crossing back and forth over 2 drips of blood turns you into a dirty hobo?

Reply:
it was mainly for testing, I'll probably make it prob based